src/github.com/getlantern/bandwidth/bandwidth.go

```diff
@@ -0,0 +1,80 @@
+package bandwidth
+
+import (
+	"net/http"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/getlantern/golog"
+)
+
+var (
+	log = golog.LoggerFor("bandwidth")
+
+	quota *Quota
+	mutex sync.RWMutex
```

Comment:
We can't fetch the data from the channel because it's not a buffered channel. Even if we made it a buffered channel, it would still be capped, so at some point the channel would be capturing stale data.